### PR TITLE
[ADD] crm: unattended leads report doc

### PR DESCRIPTION
**Pull request by jero-odoo**
Adding new document for CRM unattended leads report

### CODE REVIEW

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
+=======================
+Unattended leads report
+=======================
+
+*Unattended leads* are leads that have scheduled activities that are either due or past due.
+Whenever an activity is scheduled, *Odoo* tracks the due date and sends email reminders to the users
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:6]:
```suggestion
Whenever an activity is scheduled, Odoo tracks the due date, and sends email reminders to the users
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+   list view, they see the following:
+
+   .. image:: unattended_leads_report/unattended-leads-example.png
+      :align: center
+      :alt: List view of a sample Unattended Leads report with the activities emphasized.
+
+  Their team member, Mitchell, has two leads in the Proposition stage with activities that are due.
+  The yellow :guilabel:`📞 (phone)` icon indicates that the `5 VP Chairs` lead has a phone call
+  activity scheduled for today. The red :guilabel:`✉️ (envelope)` icon indicates that the `Modern
+  Open Space` lead has an email activity scheduled that is past due.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:28]:
```suggestion
  Their team member, Mitchell, has two leads in the *Proposition* stage with activities that are
  due.
  
  The yellow :guilabel:`📞 (phone)` icon indicates that the `5 VP Chairs` lead has a phone call
  activity scheduled for today. The red :guilabel:`✉️ (envelope)` icon indicates that the `Modern
  Open Space` lead has an email activity scheduled that is past due.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Click the first field for the new rule, and type `Activities` in the search bar, or scroll to search
+through the list. Then, next to :guilabel:`Activities`, click the arrow to open a new drop-down menu
+with secondary conditions. Select :guilabel:`Due Date`.
+
+  .. image:: unattended_leads_report/activities-due.png
+     :align: center
+     :alt: Custom filter pop-up with emphasis on the options for activities and due date.
+
+In the rule's second field, select :guilabel:`<=` from the drop-down menu. Selecting this filter
+includes all activities with a due date up to, and including the date selected in the next field.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:78]:
```suggestion
includes all activities with a due date up to, and including, the date selected in the next field.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+Either leave the rule's third field as today's date, or adjust it as needed.
+
+Exclude unassigned leads
+------------------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Salesperson` in the
+search bar, or scroll to search through the list.
+
+In the rule's second field, select :guilabel:`is set` from the drop-down menu. Selecting this filter
+excludes any leads that have note been assigned to a specific salesperson.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:89]:
```suggestion
excludes any leads that have not been assigned to a specific salesperson.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+field.
+
+.. note::
+   This filter is optional. To view results for the entire company, do **not** add this filter.
+
+.. figure:: unattended_leads_report/configured-custom-rules.png
+   :align: center
+   :alt: An example of the Custom Filter pop-up window with all of the rules configured.
+
+   An example of the Add Custom Filter pop-up window with all of the rules configured.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:109]:
```suggestion
   An example of the :guilabel:`Add Custom Filter` pop-up window with all of the rules configured.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
+assigned to a salesperson where an activity is due on the current date, or is past due. The default
+display is a bar graph, where the leads are grouped by *stage*.
+
+To group the results by salesperson, click the down arrow to the right of the search bar. Under the
+:guilabel:`Group By` heading, select :guilabel:`Salesperson`.
+
+.. note::
+   The option to group by :guilabel:`Sales Team` is available under the :guilabel:`Group By` heading
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:122]:
```suggestion
   The option to group by :guilabel:`Sales Team` is available under the :guilabel:`Group By` heading,
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+.. _track_leads/unattended_leads_report/activity tracking:
+
+Activity tracking
+=================
+
+:doc:`Activities </applications/productivity/discuss/overview/plan_activities>` are follow-up tasks
+tied to a record in the database. Activities can be scheduled on any page of the database that
+contains a chatter thread, Kanban view, list view, or activities view of an application.
+
+A best practice for pipeline management is scheduling activities or marking them as *done* anytime a
```
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:151]:
```suggestion
A best practice for pipeline management is scheduling activities, or marking them as *done*, anytime a
```

**Review by ksc-odoo — APPROVED** [review]
hey @jero-odoo -- just finished my Peer Review of this doc -- awesome job. Approving now, because you only need to address a few, super minor things. Once those are done, I think you can move this on to the next stage. Nicely done! 👍

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+   For more information, refer to :ref:`Activity Tracking
+   <track_leads/unattended_leads_report/activity tracking>`
+
+Create an unattended leads report
+=================================
+
+To create an unattended leads report, first navigate to :menuselection:`CRM app --> Reporting -->
+Pipeline` to open the :guilabel:`Pipeline Analysis` report. Click into the search bar at the top of
+the page. The :guilabel:`Created on` filter can remain active, though if any other filters are
+present, they should be removed.
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:51]: - It's not *quite* a report yet (it's a *dashboard* where reports are generated)
- let's label the search bar with a gui and the placeholder text it's been given since it's now part of an active instruction.
- add transitional language
- wording adjustment

```suggestion
Pipeline` to open the :guilabel:`Pipeline Analysis` dashboard. Then, click into the
:guilabel:`Search...` bar at the top of the page, and remove all of the default filters. Note, the
:guilabel:`Created on` filter may remain active, as this variable can be useful to include in the
report.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+Click the down arrow to the right of the search bar to open the filters drop-down menu. Click
+:guilabel:`Add Custom Filter`. This opens a :guilabel:`Add Custom Filter` pop-up window.
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:54]: This section is way too curt and does not _guide_ users or adequately describe what they're clicking through. Specifically, we're missing: 
- emoji 🔻
- form coverage / depth of content, 
- gui(s) (🔻button, search bar vs. **Search...** bar)
- grammar (e.g. a vs. an)
- transitional language (first, next, then, lastly, etc.)

```suggestion
Next, add custom filters by clicking the :guilabel:🔻 (triangle pointed down) icon to the right of
the :guilabel:`Search...` bar to open the drop-down mega menu that contains :guilabel:`Filters`,
:guilabel:`Group By`, and :guilabel:`Favourites` columns. Under the :guilabel:`Filters` column, click
:guilabel:`Add Custom Filter`, which opens an :guilabel:`Add Custom Filter` pop-up window.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @jero-odoo 

Pausing here— I have some comments for you below.

Overall I'd like to see deeper UI coverage along with a more elevated presentation of the content so we're adequately covering the dashboards + dashboard options, using very specific language, terminology, and icons, and not including any markup or grammar issues. 

Given how sensitive CRM fields and filters are, it's really important that we cover everything seen on the UI and guide users clearly and in a way that also teaches them how to "think" in Odoo (vs. just following some directions blindly).

Try to expand this article using an _ELI5_ approach, using the suggestions I have below as a guide. Yes, it's more writing, but I think it's worth it since it includes so much more context and instructional queues to improve reader comprehension.

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+----------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
+operator limits results to the sales teams selected in the next field.
+
+In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
+selected in this field.
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:106]: This is cool 👍 I didn't know you could add multiple teams here and it would count the parameters as part of an OR argument. I thought you had to create an **ANY** branch with 2+ choices in order to do this.

Suggestion to specify further what adding multiple teams in this field means, logically:
```suggestion
In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
added in this field, where each parameter will be treated with an "or" (e.g. "any") operator in the search logic.
```

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+
+.. image:: unattended_leads_report/all-custom-filter.png
+   :align: center
+   :alt: Example of overdue activities notification in the chatter of a lead.
+
+After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
+assigned to a salesperson where an activity is past due, or is due on the current date. The default
+display is a bar graph, where the leads are grouped by *stage*.
+
+To group the results by salesperson, click the down arrow to the right of the search bar. Under the
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:133]: missing guis here I think?

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+:guilabel:`Group By` heading, select :guilabel:`Salesperson`.
+
+.. note::
+   The option to group by :guilabel:`Sales Team` is also available under the :guilabel:`Group By`
+   heading.
+
+To change to a *list* view, click the :guilabel:`≣ (bars)` icon in the top-right corner of the
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:140]: Is **(bars)** what that icon is actually called? 
Hint 👇 
Screenshot 2024-03-11 at 3 08 31 PM

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+screen.
+
+.. tip::
+   Clicking the :guilabel:`(sliders) icon` opens a drop-down menu of additional columns that can be
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:144]: Same here for **(sliders)**
Screenshot 2024-03-11 at 3 11 09 PM

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+   - :guilabel:`Expected Closing`: the estimated date on which the lead will be won.
+   - :guilabel:`Probability`: estimated success rate based on the stage.
+
+  .. image:: unattended_leads_report/additional-options.png
+     :align: center
+     :alt: Custom filter pop-up with emphasis on the options for activities and due date.
+
+.. _track_leads/unattended_leads_report/activity tracking:
+
+Activity tracking
```
Comment (StraubCreative) [inline content/applications/sales/crm/track_leads/unattended_leads_report.rst:159]: Not sure this section is necessary since the [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc already exists and was pointed to much earlier in the article. 

Replacement ideas:
- a **seealso** admonition pointing to [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc (again, as a redundancy)
- best practices for sales team success if they want to implement activities tracking for their team.
- just leave blank and just remove what's there (can always add more later if we want)

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @jero-odoo 
Good revisions last round. I have a few more to get us to the finish line.
Please let me know if you have any questions or want to discuss specific points further, thank you!

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst (hunk 13 above)
Comment (jero-odoo) [reply content/applications/sales/crm/track_leads/unattended_leads_report.rst:145]: (FYI, it won't let me view these images, it says they are from a "private user", but I was able to figure out what you were trying to hint at :smiley: )

File: content/applications/sales/crm/track_leads/unattended_leads_report.rst (hunk 14 above)
Comment (jero-odoo) [reply content/applications/sales/crm/track_leads/unattended_leads_report.rst:160]: I think just the see also is best. At some point, it will probably be worth it to do one "Activities in the CRM" doc, which I know we discussed in the past, that can be linked here. Currently, the activities doc is about the function of the feature, but if we want to go into how to utilize them within the CRM best specifically, I think it would be best to keep that in one separate doc, instead of adding that on as a footnote. Thoughts?

Comment (StraubCreative) [reply content/applications/sales/crm/track_leads/unattended_leads_report.rst]: > Thoughts?
Agree! I'll make a task.

**Review by StraubCreative — APPROVED** [review]
Well done @jero-odoo this is a great doc 👍 
.....
@robodoo r+
